package.json

```diff
@@ -13,7 +13,8 @@
         "lint-fix": "run foreach-with-fallback lint-fix global:lint-fix",
         "fmt-check": "run foreach-with-fallback fmt-check global:fmt-check",
         "fmt-fix": "run foreach-with-fallback fmt-fix global:fmt-fix",
-        "test": "NODE_ENV=test jest && yarn workspace open-discussions run test",
+        "test": "yarn global:test",
+        "test-watch": "yarn global:test-watch",
```

Comment:
- **Previously**, `yarn test` ran the tests in "old open" and "new open". 
- **Now**: `yarn test` and `yarn test-watch` run tests in "new open" only. *You can still run the old tests via `yarn workspace open-discussions test`, which is what CI does.

My rationale here is that:
1. Mostly we are changing the new code, so most of the time this is what we want.
2. Also, using `yarn test` and `yarn test-watch` to run the new tests is much more discoverable / intuitive than running things like `yarn workspace infinite-corridor global:test-watch`. **And running `yarn test-watch` is just as performant locally; just `p` to focus.**.